Basic-Car-Maintenance/Shared/Dashboard/Views/DashboardView.swift

```diff
@@ -50,6 +50,13 @@ struct DashboardView: View {
                     
                 }
             }
+            .alert("Failed To Delete Event", isPresented: $viewModel.showErrorAlert, actions: {
+                Button("OK") {
+                    viewModel.showErrorAlert = false
+                }
+            }, message: {
+                Text(viewModel.errorDeleting?.localizedDescription ?? "Error while deleting event").padding()
```

Comment:
As noted above just pass the localized description, and handle the optional in the view model rather than here in the view
(This would mean the `errorDeleting` would actually end up being a `String` rather than `Error`)

Reply:
Got it! Pushed the code with the suggested changes. Let me know if the code needs more changes.